extensions/dicom-sr/src/utils/getToolStateToCornerstoneMeasurementSchema.js

```diff
@@ -174,6 +245,15 @@ function ArrowAnnotate(measurementData, imageId, _getValueTypeFromToolType) {
     StudyInstanceUID,
   } = instance;
 
+  const displaySets = DisplaySetService.getDisplaySetsForSeries(
+    SeriesInstanceUID
+  );
+  const displaySet = displaySets.find(ds => {
+    return (
+      ds.images && ds.images.some(i => i.SOPInstanceUID === SOPInstanceUID)
+    );
+  });
+
```

Comment:
This bit appears to be reused and may be worth pulling out

Reply:
Done in next commit.